src/PersistenceProviders/Proto.Persistence.Sqlite/SqliteProvider.cs

```diff
@@ -1,22 +1,41 @@
 ﻿using System;
 using System.Threading.Tasks;
-using System.Linq;
 using Newtonsoft.Json;
+using Microsoft.Data.Sqlite;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Proto.Persistence.Sqlite
 {
     public class SqliteProvider : IProvider
     {
         private readonly string _datasource;
+        private string _connectionString => $"{new SqliteConnectionStringBuilder { DataSource = _datasource }}";
 
         public SqliteProvider(string datasource = "states.db")
```

Comment:
Maybe add an overload to accept a SqliteConnectionStringBuilder as providing a custom string with different sqlite settings is something I'd want to do